Error messages, from fastai20 env, "jupyter notebook" shell command.  
Warning, but server still works, opens notebooks with default Python 3 engine. 

---   

(fastai20) jyoon@Asus-S510:~/work$ jupyter notebook                                                                     [W 21:44:26.910 NotebookApp] Error loading server extension voila.server_extension                                          Traceback (most recent call last):                                                                                        File "/home/jyoon/conda3/envs/fastai20/lib/python3.6/site-packages/notebook/notebookapp.py", line 1991, in init_server_extensions                                                                                                                 func(self)                                                                                                            File "/home/jyoon/conda3/envs/fastai20/lib/python3.6/site-packages/voila/server_extension.py", line 44, in _load_jupyter_server_extension                                                                                                         template_paths = collect_template_paths(['voila', 'nbconvert'], template_name)                                        File "/home/jyoon/conda3/envs/fastai20/lib/python3.6/site-packages/voila/paths.py", line 24, in collect_template_paths                                                                                                                            return collect_paths(app_names, template_name, include_root_paths=True, prune=prune, root_dirs=root_dirs)             File "/home/jyoon/conda3/envs/fastai20/lib/python3.6/site-packages/voila/paths.py", line 53, in collect_paths             root_dirs = root_dirs or _default_root_dirs()                                                                         File "/home/jyoon/conda3/envs/fastai20/lib/python3.6/site-packages/voila/paths.py", line 102, in _default_root_dirs                                                                                                                               if nbconvert.exporters.templateexporter.DEV_MODE:                                                                   AttributeError: module 'nbconvert.exporters.templateexporter' has no attribute 'DEV_MODE'                           [I 21:44:26.913 NotebookApp] Serving notebooks from local directory: /home/jyoon/work                                   [I 21:44:26.913 NotebookApp] Jupyter Notebook 6.2.0 is running at:                                                      [I 21:44:26.913 NotebookApp] http://localhost:8888/?token=888c06192dd7b74730e22333e0939ea476080a29da068874              [I 21:44:26.914 NotebookApp]  or http://127.0.0.1:8888/?token=888c06192dd7b74730e22333e0939ea476080a29da068874          [I 21:44:26.914 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).  [C 21:44:26.920 NotebookApp]                                                                                                                                                                                                                        To access the notebook, open this file in a browser:                                                                        file:///home/jyoon/.local/share/jupyter/runtime/nbserver-1256-open.html                                             Or copy and paste one of these URLs:                                                                                        http://localhost:8888/?token=888c06192dd7b74730e22333e0939ea476080a29da068874                                        or http://127.0.0.1:8888/?token=888c06192dd7b74730e22333e0939ea476080a29da068874                                   Unable to init server: Broadway display type not supported: localhost:0.0                                               Error: cannot open display: localhost:0.0                       (fastai20) jyoon@Asus-S510:~/work$ jupyter notebook                                                                     [W 21:44:26.910 NotebookApp] Error loading server extension voila.server_extension                                          Traceback (most recent call last):                                                                                        File "/home/jyoon/conda3/envs/fastai20/lib/python3.6/site-packages/notebook/notebookapp.py", line 1991, in init_server_extensions                                                                                                                 func(self)                                                                                                            File "/home/jyoon/conda3/envs/fastai20/lib/python3.6/site-packages/voila/server_extension.py", line 44, in _load_jupyter_server_extension                                                                                                         template_paths = collect_template_paths(['voila', 'nbconvert'], template_name)                                        File "/home/jyoon/conda3/envs/fastai20/lib/python3.6/site-packages/voila/paths.py", line 24, in collect_template_paths                                                                                                                            return collect_paths(app_names, template_name, include_root_paths=True, prune=prune, root_dirs=root_dirs)             File "/home/jyoon/conda3/envs/fastai20/lib/python3.6/site-packages/voila/paths.py", line 53, in collect_paths             root_dirs = root_dirs or _default_root_dirs()                                                                         File "/home/jyoon/conda3/envs/fastai20/lib/python3.6/site-packages/voila/paths.py", line 102, in _default_root_dirs                                                                                                                               if nbconvert.exporters.templateexporter.DEV_MODE:                                                                   AttributeError: module 'nbconvert.exporters.templateexporter' has no attribute 'DEV_MODE'                           [I 21:44:26.913 NotebookApp] Serving notebooks from local directory: /home/jyoon/work                                   [I 21:44:26.913 NotebookApp] Jupyter Notebook 6.2.0 is running at:                                                      [I 21:44:26.913 NotebookApp] http://localhost:8888/?token=888c06192dd7b74730e22333e0939ea476080a29da068874              [I 21:44:26.914 NotebookApp]  or http://127.0.0.1:8888/?token=888c06192dd7b74730e22333e0939ea476080a29da068874          [I 21:44:26.914 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).  [C 21:44:26.920 NotebookApp]                                                                                                                                                                                                                        To access the notebook, open this file in a browser:                                                                        file:///home/jyoon/.local/share/jupyter/runtime/nbserver-1256-open.html                                             Or copy and paste one of these URLs:                                                                                        http://localhost:8888/?token=888c06192dd7b74730e22333e0939ea476080a29da068874                                        or http://127.0.0.1:8888/?token=888c06192dd7b74730e22333e0939ea476080a29da068874                                   Unable to init server: Broadway display type not supported: localhost:0.0                                               Error: cannot open display: localhost:0.0                       
